src/core/Akka.FSharp/FsApi.fs

```diff
@@ -316,21 +328,21 @@ module Strategy =
         upcast OneForOneStrategy(Nullable retries, Nullable timeout, System.Func<_, _>(decider))
     
     /// <summary>
-    /// Returns a supervisor strategy appliable only each supervised actor when any of them had faulted during execution.
+    /// Returns a supervisor strategy appliable to each supervised actor when any of them had faulted during execution.
     /// </summary>
     /// <param name="decider">Used to determine a actor behavior response depending on exception occurred.</param>
     let allForOne (decider : exn -> Directive) : SupervisorStrategy = 
         upcast AllForOneStrategy(System.Func<_, _>(decider))
     
     /// <summary>
-    /// Returns a supervisor strategy appliable only each supervised actor when any of them had faulted during execution.
+    /// Returns a supervisor strategy appliable to each supervised actor when any of them had faulted during execution.
     /// </summary>
     /// <param name="retries">Defines a number of times, an actor could be restarted. If it's a negative value, there is not limit.</param>
     /// <param name="timeout">Defines time window for number of retries to occur.</param>
     /// <param name="decider">Used to determine a actor behavior response depending on exception occurred.</param>
     let allForOne2 (retries : int) (timeout : TimeSpan) (decider : exn -> Directive) : SupervisorStrategy = 
```

Comment:
I believe the F# API should use static members with optional arguments in cases like this.  (An F# API still looks valuable for it's use of F# function types, and removing the subtyping).

Note that F# API design can and should use members when appropriate, especially for metadata-rich APIs (think of List.map as "low metadata" and FSHarp.Charting or this API as "high metadat"). Indeed there are many elements of the language relevant to API design such as optional arguments which are only supported on members.